### PR TITLE
docs: changes in Spanish translations

### PR DESCRIPTION
**Pull request by gboquizosanchez**
### Summary 
- Almost all the general pages have been added.
- Some pages have been retranslated or fixed.
- Other minor fixes.

### CODE REVIEW

File: docs/general/faq.md
Comment (wescopeland) [inline docs/general/faq.md:1]: Thanks for fixing these typos!

File: docs/general/rabot.md
Comment (wescopeland) [inline docs/general/rabot.md:1]: Good catch.

File: docs/es/developer-docs/achievement-scoring.md
Comment (wescopeland) [inline docs/es/developer-docs/achievement-scoring.md:1]: Any reason this English page should be in the Spanish docs?

**Review by wescopeland — COMMENTED** [review]
Really looks great. Thanks so much for taking the time to work on this. I only have one question and once it's resolved I think we're good.

File: docs/general/faq.md
Comment (gboquizosanchez) [reply docs/general/faq.md:1]: You're welcome, I just passed through hahah

File: docs/es/developer-docs/achievement-scoring.md
Comment (gboquizosanchez) [reply docs/es/developer-docs/achievement-scoring.md:1]: Probably It was a momenteous workaround because vitepress was crashing the app, let me fix it.

Comment (gboquizosanchez) [reply docs/es/developer-docs/achievement-scoring.md:1]: Fixed!

**Review by wescopeland — APPROVED** [review]
Thank you!

**Pull request merged**